tests/playwright/src/specs/image-smoke.spec.ts

```diff
@@ -64,6 +66,31 @@ test.describe.serial('Image workflow verification @smoke', () => {
     playExpect(await updatedImages.getCurrentStatusOfImage(helloContainer)).toBe('UNUSED');
   });
 
+  test('Pull image from search results', async ({ navigationBar }) => {
```

Comment:
This seems like a good start. I can imagine very many other options we would need to cover.